src/NServiceBus.Core/Unicast/Behaviors/LoadHandlersConnector.cs

```diff
@@ -20,9 +20,7 @@ public override void Invoke(LogicalMessageProcessingStageBehavior.Context contex
         {
             bool callbackInvoked;
 
-
-            // for now we cheat and pull it from the behavior context:
-            if (!context.TryGet(CallbackInvocationBehavior.CallbackInvokedKey, out callbackInvoked))
+            if (!context.TryGet("NServiceBus.CallbackInvocation.CallbackWasInvoked", out callbackInvoked))
             {
                 callbackInvoked = false;
```

Comment:
yeah, that could help solidify the contract between the core and other components, feel free to change it